main/src/cgeo/geocaching/connector/trackable/GeokretyAuthorizationActivity.java

```diff
@@ -0,0 +1,77 @@
+package cgeo.geocaching.connector.trackable;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.activity.TokenAuthorizationActivity;
+import cgeo.geocaching.network.Network;
+import cgeo.geocaching.settings.Settings;
+
+import ch.boye.httpclientandroidlib.HttpResponse;
+
+import android.os.Bundle;
+
+import java.util.regex.Pattern;
+
+public class GeokretyAuthorizationActivity extends TokenAuthorizationActivity {
+
+    private static final Pattern PATTERN_IS_ERROR = Pattern.compile("error ([\\d]+)");
```

Comment:
Are you sure?
This pattern is a specific check for the GK responses from server.
Those types of constant shouldn't be "local", as for strings?